configs/containers/specs/jedi-ci.yaml

```diff
@@ -12,10 +12,10 @@
     ecbuild@3.7.2,
     eccodes@2.33.0,
     ecflow@5.11.4,
-    eckit@1.27.0,
-    ecmwf-atlas@0.38.1 +fckit +trans +tesselation +fftw,
-    fiat@1.2.0,
-    ectrans@1.2.0 +fftw,
+    eckit@1.28.3,
+    ecmwf-atlas@0.40.0 +fckit +trans +tesselation +fftw,
+    fiat@1.4.1,
+    ectrans@1.5.0 +fftw,
     eigen@3.4.0,
     fckit@0.11.0,
```

Comment:
@stiggy87 I think the reason the container builds are failing is because we overlooked updating `fckit` in this file.

Reply:
Whoops! Definitely an oversight on my part. Getting a bugfix PR created for this. Thanks for pointing it out!